android/app/src/test/java/com/microsoft/codepush/common/CodePushConfigurationTest.java

```diff
@@ -0,0 +1,67 @@
+package com.microsoft.codepush.common;
+
+import com.microsoft.appcenter.utils.AppCenterLog;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static com.microsoft.codepush.common.TestUtils.APP_VERSION;
+import static com.microsoft.codepush.common.TestUtils.CLIENT_UNIQUE_ID;
+import static com.microsoft.codepush.common.TestUtils.DEPLOYMENT_KEY;
+import static com.microsoft.codepush.common.TestUtils.PACKAGE_HASH;
+import static com.microsoft.codepush.common.TestUtils.SERVER_URL;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.times;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.verifyStatic;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({AppCenterLog.class})
+public class CodePushConfigurationTest {
+    @Before
+    public void setUp() {
+        mockStatic(AppCenterLog.class);
+    }
+
+    @Test
+    public void correctConfigurationTest() {
+        CodePushConfiguration correctConfig = new CodePushConfiguration();
+        correctConfig.setAppVersion(APP_VERSION)
```

Comment:
It is a bad practice to share constant values between tests files. Each test should be isolated from each other. All constants references like `APP_VERSION`, `CLIENT_UNIQUE_ID` and so on should be replaced with hardcoded values. Please fix it here and in another test files.